config/clients/python/template/README_calling_api.mustache

```diff
@@ -174,38 +174,58 @@ Reads the relationship tuples stored in the database. It does not evaluate nor e
 
 ```python
 # Find if a relationship tuple stating that a certain user is a viewer of certain document
-body = TupleKey(
+body = ReadRequest(
```

Comment:
Should this be `ClientReadRequest`?

Reply:
I have copied and pasted the changes from https://github.com/openfga/python-sdk/pull/31/files
Actually...it should be TupleKey as noted in https://github.com/openfga/python-sdk/blob/4a031397970ee1faca19d76e7edc5ca12511e630/openfga_sdk/client/client.py#L332

Reply:
We will need to update the README in the generated SDK to reflect this